### PR TITLE
integrate bounty runtime

### PR DESCRIPTION
**Pull request by 4meta5**
- [x] runtime
- [x] node
- [ ] client
- [ ] cli
- [ ] github actions ci

### CODE REVIEW

File: runtime/src/lib.rs [hunk 1]
```diff
@@ -251,6 +251,33 @@ impl sunshine_faucet_pallet::Trait for Runtime {
...
+    type VoteId = u64;
+    type Signal = u64;
+}
+
+parameter_types! {
+    pub const MinimumDisputeAmount: u64 = 10;
+}
+impl sunshine_court::Trait for Runtime {
+    type Event = Event;
+    type Currency = u64;
```
Comment (4meta5) [inline runtime/src/lib.rs:276]: The error happens because of this line. Regardless of what I set that Currency type to

Comment (4meta5) [reply runtime/src/lib.rs:276]: When I take out the impl of the court trait, there are no errors

Comment (4meta5) [reply runtime/src/lib.rs:276]: and if I add any of the bounty modules that use the Currency trait, I get the same error. These modules use that trait to use the Balances type so it is not optional

File: runtime/Cargo.toml [hunk 2]
```diff
@@ -37,6 +37,13 @@ sp-version = { version = "2.0.0-rc4", default-features = false }
 sunshine-faucet-pallet = { version = "0.1.0", default-features = false }
 sunshine-identity-pallet = { version = "0.1.0", default-features = false }
 sunshine-identity-utils = { version = "0.1.0", default-features = false }
+sunshine-util = { version = "0.0.6", default-features=false}
+sunshine-org = { version = "0.0.6", default-features=false }
+sunshine-vote = { version = "0.0.6", default-features=false}
+sunshine-court = { version = "0.0.6", default-features=false}
```
Comment (dvc94ch) [inline runtime/Cargo.toml:43]: formatting, `default-features = false }` and in case you noticed the modules are ordered alphabetically

Comment (4meta5) [reply runtime/Cargo.toml:43]: _[bikeshedding](https://en.wikipedia.org/wiki/Law_of_triviality)_

File: node/src/chain_spec.rs [hunk 3]
```diff
@@ -235,6 +253,12 @@ fn testnet_genesis(
                 .map(|x| (x.3.clone(), 1))
                 .collect(),
         }),
+        pallet_sudo: Some(SudoConfig { key: root_key }),
```
Comment (4meta5) [inline node/src/chain_spec.rs:256]: @dvc94ch sudo is useful when you start these networks for clean runtime upgrades, once we've done this with a sudo, then we can gradually experiment with more democratic chain governance (like Polkadot)

**Comment by 4meta5** [comment]
Is there a reason we haven't added a CI to this repo?

**Comment by shekohex** [comment]
> Is there a reason we haven't added a CI to this repo?

Use Github actions from [sunshine-identity](https://github.com/sunshine-protocol/sunshine-identity) or [dart-bindgen](https://github.com/sunshine-protocol/dart-bindgen)

**Comment by 4meta5** [comment]
starting over again I guess, easier than rebasing changes on top of master

**Pull request closed without merging**